test/controllers/EventsControllerSpec.scala

```diff
@@ -55,19 +55,21 @@ class EventsControllerSpec extends UnitWithAppSpec {
     }
 
     "return OK with all events" in new TestFixture {
```

Comment:
@kjharkin Does it need to say "and an empty List" since the test is only checking the status and not the json body?